ehcache-core/src/main/java/org/ehcache/core/spi/store/tiering/AuthoritativeTier.java

```diff
@@ -75,7 +76,24 @@ public interface AuthoritativeTier<K, V> extends Store<K, V> {
    */
   void setInvalidationValve(InvalidationValve valve);
 
+
   /**
+   * Bulk method to compute a value for every key passed in the {@link Iterable} <code>keys</code> argument using the <code>mappingFunction</code>
+   * to compute the value.
+   * <p>
+   * The function gets an {@link Iterable} of {@link java.util.Map.Entry} key/value pairs, where each entry's value is its currently stored value
+   * for each key that is not mapped in the store. It is expected that the function returns an {@link Iterable} of {@link java.util.Map.Entry}
+   * key/value pairs containing an entry for each key that was passed to it. This returned {@link Iterable} should also iterate in the same order as the input {@link Iterable}.
+   * Note: This method guarantees atomicity of computations for each individual key in {@code keys}. Implementations may choose to provide coarser grained atomicity.
+   *
+   * @param keys the keys to compute a new value for, if they're not in the store.
+   * @param mappingFunction the function that generates new values.
+   * @return a {@link Map} of key/value pairs for each key in <code>keys</code> to the previously missing value.
```

Comment:
No need to `@link` a JDK type this common... plus given how the javadoc is currently generated a correct link will not be generated.

Reply:
Its looks good using `@link`, rather than simply mentioning a Map, I am not  changing this